File: web/src/model/victron_controller/dashboard/from_0_2_0_wsclientmessage_ping.ts
// @ts-nocheck
import {Ping as v0_2_0_Ping} from './v0_2_0/WsClientMessage'
import {Ping as dashboard_Ping} from './WsClientMessage'

export function convert__ws_client_message__ping__from__0_2_0(from: v0_2_0_Ping): dashboard_Ping {
    return new dashboard_Ping (
        JSON.parse(JSON.stringify(from.body))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_wsclientmessage_ping.rs


pub fn convert__wsclientmessage__ping__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::ws_client_message::Ping) -> crate::victron_controller::dashboard::ws_client_message::Ping {
    crate::victron_controller::dashboard::ws_client_message::Ping {
        body: serde_json::from_value(serde_json::to_value(&from.body).unwrap()).unwrap(),
    }
}